src/app/components/add-card/add-card.component.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AddCardComponent } from './add-card.component';
+
+describe('AddCardComponent', () => {
+  let component: AddCardComponent;
+  let fixture: ComponentFixture<AddCardComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [AddCardComponent],
+    });
+    fixture = TestBed.createComponent(AddCardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Let's add more tests. This one only tests if the `AddCardComponent` loads.

Reply:
added but the new location is card-item feature module